src/test/resources/integrationTest/integrationTest.properties

```diff
@@ -58,6 +58,7 @@ opportunityExistField=externalID
 placementChangeRequestExistField=customText1
 placementCommissionExistField=migrateGUID
 placementExistField=customText1
+placementShiftSetExistField=externalID,placement.customText1
```

Comment:
Same here - This should probably only be externalID, to keep things simple. There should be no need for a multi-field lookup.